### ISSUE TITLE
Cannot start  scanner for 'sending' tab

### ISSUE THREAD

**Issue by liorpesoa** [z-classic/electrum-zcl#37]
![image](https://user-images.githubusercontent.com/10183066/35389112-190aa54e-01df-11e8-9435-6aa3b67be205.png)

**Comment by KaboomFox** [comment]
I think we need to make a requirements.txt, a Brewfile and a pkglist to prevent this from happening. I'll start putting this together tonight.

**Comment by KaboomFox** [comment]
I was able to fix the zbar issue by 
`$ brew install zbar
$ export LDFLAGS="-L$(brew --prefix zbar)/lib"
$ export CFLAGS="-I$(brew --prefix zbar)/include"`

but the device list comes up empty because the path is hardcorded as `device_root = "/sys/class/video4linux"` which doesn't exist on a mac. You cannot access the camera on a mac via a path. https://github.com/BTCP-community/electrum-zcl/blob/master/lib/qrscanner.py#L69

I found this pr: https://github.com/spesmilo/electrum/pull/3485
which works mostly when launched from `./electrum`. Minor bugs.

 However, I cannot get it to work when laughed from a dmg. The module zbarlight cannot be found. I think this is due to the flags not in the standalone, because zbarlight is in the bundle.

For now maybe we should disable the qrscanner on macs (remove the button)? I don't think many people will be using that feature on their macbooks.

**Comment by liorpesoa** [comment]
@KaboomFox yes. let's disable. we should think MVP :) we need to ship this.

**Comment by ch4ot1c** [comment]
I vote Yes to removing the mac qr scanner temporarily.

**Comment by nimbosa** [comment]
with permanent fix pending, this is temporarily disabled on MacOS: https://github.com/z-classic/electrum-zcl/pull/64
